control-userlayer/src/main/java/org/oskari/control/userlayer/EditUserLayerHandler.java

```diff
@@ -37,11 +35,9 @@ public void init() {
     public void handlePost(ActionParameters params) throws ActionException {
         String mapSrs = params.getHttpParam(ActionConstants.PARAM_SRS);
         final UserLayer userLayer = UserLayerHandlerHelper.getUserLayer(userLayerDbService, params);
-        userLayer.setName(PropertyUtil.getDefaultLanguage(), params.getRequiredParam(PARAM_NAME)); // FIXME: userLayer.setLocale(locale);
-        userLayer.setLayer_desc(params.getHttpParam(PARAM_DESC, userLayer.getLayer_desc()));
-        userLayer.setLayer_source(params.getHttpParam(PARAM_SOURCE, userLayer.getLayer_source()));
+        userLayer.setLocale(params.getHttpParamAsJSON(PARAM_LOCALE));
```

Comment:
Not that getRequiredParam can't be used per field we should check that there's a name for the layer inside locale.